test/combineReducers.spec.js

```diff
@@ -18,14 +18,30 @@ describe('Utils', () => {
       expect(s2).toEqual({ counter: 1, stack: [ 'a' ] })
     })
 
-    it('ignores all props which are not a function', () => {
+    it('warns on props which are not a function and excludes them', () => {
+      const preSpy = console.error
+      const spy = jest.fn()
+      console.error = spy
+
       const reducer = combineReducers({
         fake: true,
         broken: 'string',
         another: { nested: 'object' },
         stack: (state = []) => state
       })
 
+      expect(spy.mock.calls[0][0]).toMatch(
```

Comment:
Another (less-precise, but terser) way to do this is

```js
expect(spy).toHaveBeenCalledWith('Reducer provided for "fake" is not a function')
```

See: https://devhints.io/jest

Reply:
Thank you! :)